backend/models.py

```diff
@@ -0,0 +1,82 @@
+from backend import db
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    public_id = db.Column(db.String(50), unique=True, nullable=False)
+    email = db.Column(db.String(50), unique=True, nullable=False)
+    username = db.Column(db.String(50), nullable=False)
+    password_hash = db.Column(db.String(80), nullable=False)
+    admin = db.Column(db.Boolean, default=False)
+    biography = db.Column(db.String(100))
+    avatar = db.Column(db.String)  # will be a filepath
+    verified = db.Column(db.Boolean, default=False, nullable=False)
+    flag_count = db.Column(db.SmallInteger, nullable=False, default=0)
+    date_created = db.Column(db.TIMESTAMP, server_default=db.func.now(), nullable=False)
+
+    def __repr__(self):
+        return '<User {email} {username}>'.format(email=self.email, username=self.username)
+
+
+class Project(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(50), nullable=False)
+    short_description = db.Column(db.String(500), nullable=False)
+    project_plan = db.Column(db.Text, nullable=False)
+    # category = db.Column(db.String(50))
+    # progress can be calculated from the target and current budget
+    # progress = db.Column(db.SmallInteger, nullable=False)  # in percentage
+    target_budget = db.Column(db.Integer, nullable=False)
+    current_budget = db.Column(db.Integer, default=0, nullable=False)
+    donators = db.Column(db.Integer, default=0, nullable=False)
+    start_date = db.Column(db.TIMESTAMP, nullable=False)
+    end_date = db.Column(db.TIMESTAMP, nullable=False)
+    cover = db.Column(db.String)
+    verified = db.Column(db.Boolean, nullable=False, default=False)
```

Comment:
Country nog weer even toevoegen, of heb je andere wilde plannen?

Reply:
Wordt die country handmatig ingevuld? ik dacht dat het misshien handiger is om alleen de coordinates op te slaan. Maar je opzich kan ik dat het er wel inhouden. Maar dan moeten ze kunnen kiezen uit een land neem ik aan? Anders kan iemand gewoon plopseland invullen? 🤣

Reply:
Ja alleen denk dat gebruikers makkelijker een land invullen dan coordinaten :P

Reply:
Daarom hebben ze een kaartje nodig om uit de kiezen of zo'n searchbar die je overal ziet. Wat gaan we doen.